spec/support/dnsimple_stubs.rb

```diff
@@ -0,0 +1,78 @@
+module DnsimpleStubs
+  BASE_URL = "https://api.dnsimple.com/v1"
+
+  def stub_successful_cname_registration(subdomain, url)
+    stub_cname_registration_request(subdomain, url).
+      to_return(
+        status: 201,
+        body: dnsimple_response_body(subdomain, url),
+      )
+  end
+
+  def stub_failed_cname_registration(subdomain, url, message = "oops")
+    stub_cname_registration_request(subdomain, url).
+      to_return(
+        status: 400,
+        body: failed_dnsimple_response_body(subdomain, message),
+      )
+  end
+
+  def stub_cname_registration_request(subdomain, url)
+    stub_request(:post, dnsimple_api_url("/records")).
+      with(
+        body: dnsimple_request_body(subdomain, url),
+        headers: dnsimple_headers
+    )
+  end
+
+  def failed_dnsimple_response_body(subdomain, message)
```

Comment:
Unused method argument - `subdomain`. If it's necessary, use `_` or `_subdomain` as an argument name to indicate that it won't be used.